bindings/c/test/mako/utils.c

```diff
@@ -80,6 +80,18 @@ void genkey(char* str, char* prefix, int prefixlen, int prefixpadding, int num,
 	str[len - 1] = '\0';
 }
 
+void updateKeyPrefix(char* str,int prefixlen, int prefixpadding, int len){
+	//Acquire a random prefix
+	const int prefixoffset = prefixpadding ? len - prefixlen - 1 : 0;
+	char* prefixstr = (char*)malloc(sizeof(char)*prefixlen+1);
```

Comment:
I'm confused. `prefixlen` is the length of the original prefix (e.g. "mako"), no?
Or is the caller supposed to set it larger than `KEYPREFIXLEN`?

Reply:
I just updated this function and will push it ASAW. Thanks!